### PR TITLE
[improve][doc] Relocate `docker-compose` content to `Get Started` category

### PR DESCRIPTION
**Pull request by momo-jun**
Fixes https://github.com/apache/pulsar/issues/19684

### Modifications

1. Add a new getting-started topic for running a Pulsar cluster with `docker-compose'.
2. Move the `docker-compose` section from https://pulsar.apache.org/docs/next/deploy-docker/ to the new topic.
3. Add the link to the home page and left navigation.

Preview looks good:
image

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: docs/develop-schema.md [hunk 1]
```diff
@@ -1,62 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (momo-jun) [reply docs/develop-schema.md:1]: Not really, but it's a duplicate page and might be added unexpectedly. The content is presented in [here](https://pulsar.apache.org/docs/next/schema-get-started/#customize-schema-storage).

File: docs/getting-started-docker-compose.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+id: getting-started-docker-compose
+title: Run a Pulsar cluster locally with `docker-compose`
+sidebar_label: "Run Pulsar locally with `docker-compose`"
```
Comment (tisonkun) [inline docs/getting-started-docker-compose.md:4]:
```suggestion
title: Run a Pulsar cluster locally with Docker Compose
sidebar_label: "Run Pulsar locally with Docker Compose"
```

Use the formal text name. See https://docs.docker.com/compose/.

**Review by tisonkun — APPROVED** [review]
Generally looks good! One comment inline.

File: docs/getting-started-docker-compose.md (hunk 2 above)
Comment (momo-jun) [reply docs/getting-started-docker-compose.md]: Thanks for pointing it out. Resolved in [b52acfa](https://github.com/apache/pulsar-site/pull/448/commits/b52acfaad99aefe158556af2bb4c442fa4ca5809).

**Review by asafm — CHANGES_REQUESTED** [review]

**Comment by asafm** [comment]
The image has a title of  "`docker compose`". Should be Docker Compose

File: docs/getting-started-home.md [hunk 3]
```diff
@@ -8,5 +8,5 @@ Getting up and running with Pulsar is simple. Download it, install it, and try i
 
 You have three options. Click any of these links to begin your Pulsar journey!
 * [Run a standalone Pulsar cluster locally](getting-started-standalone.md) - Run a single instance of Pulsar in standalone mode on a single machine.
-* [Run a standalone Pulsar cluster in Docker](getting-started-docker.md) - Run one or more instances of Pulsar in a Docker container.
+* [Run a standalone Pulsar cluster in Docker](getting-started-docker.md) - Run one or more instances of Pulsar in a Docker container. If you want to quickly set up a small Pulsar cluster on your local laptop, go to [Run a Pulsar cluster with `docker-compose`](getting-started-docker-compose.md).
```
[the remark this answers is not in the document]
Comment (momo-jun) [reply docs/getting-started-home.md:11]: Right! I missed this one. Thank you.

**Comment by momo-jun** [comment]
Thanks @jak78 for pointing out that the `docker-compose` command (with a dash) is being deprecated in favor of `docker compose` (without a dash), and the `docker-compose.yml` will be named `compose.yml`.
@liangyepianzhou It seems the original ones will not be supported by the end of June 2023. Shall we apply the change right now or document anything to help users to switch to [Compose V2](https://docs.docker.com/compose/reference/)?

**Comment by liangyepianzhou** [comment]
IMO, we should apply the change right now and there is no need to keep the deprecated version.

**Comment by momo-jun** [comment]
> IMO, we should apply the change right now and there is no need to keep the deprecated version.

Thank you. As discussed, @liangyepianzhou will help verify the cmd change and I will update the doc accordingly.

**Comment by liangyepianzhou** [comment]
I have tested this `docker-compose.yml` file with the new `docker compose`.
After applying the changes, the cluster can be properly deployed.

**Review by liangyepianzhou — APPROVED** [review]

**Review by jak78 — APPROVED** [review]

**Pull request merged**